config/config_test.go

```diff
@@ -190,21 +191,31 @@ func TestLoadConfigFile(t *testing.T) {
 		t.Error("PostMetricsRetryMax should be 5")
 	}
 
-	if config.Plugin["metrics"] == nil {
+	if config.MetricPlugins == nil {
 		t.Error("plugin should have metrics")
 	}
-	pluginConf := config.Plugin["metrics"]["mysql"]
+	pluginConf := config.MetricPlugins["mysql"]
 	if pluginConf.Command != "ruby /path/to/your/plugin/mysql.rb" {
 		t.Errorf("plugin conf command should be 'ruby /path/to/your/plugin/mysql.rb' but %v", pluginConf.Command)
 	}
 	if pluginConf.User != "mysql" {
 		t.Errorf("plugin user_name should be 'mysql'")
 	}
+	if *pluginConf.CustomIdentifier != "app1.example.com" {
+		t.Errorf("plugin custom_identifier should be 'app1.example.com'")
```

Comment:
it should be `t.Error` or show `*pluginConf.CustomIdentifier` as well.
Including following error messages.